### ISSUE TITLE
Add disk io speed & bandwith tracffic usage limit &  bandwith speed limit

### ISSUE THREAD

**Issue by anime777** [canonical/multipass#3278]
1.  disk io speed  limit

2. & bandwith tracffic usage limit 

3. &  bandwith speed limit

Does anybody know how to do this?

**Comment by townsend2010** [comment]
Hi @anime777!

I'm a little unsure what you are asking for here.  Are you asking if there are ways to tune Multipass to limit disk IO speed, limit the network traffic usage and limit the network speed limit?

**Comment by anime777** [comment]
Yes,   Are there  ways to do that?

**Comment by townsend2010** [comment]
No, not within Multipass, and i assume you are wanting this for each instance that is launched which would be very complicated to do.

**Comment by jimporter** [comment]
At least on qemu, we could pass in the appropriate flags to do this: https://raw.githubusercontent.com/qemu/qemu/refs/heads/master/docs/throttle.txt
